CodeEdit/Features/Settings/Pages/ThemeSettings/Models/ThemeModel.swift

```diff
@@ -120,91 +124,106 @@ final class ThemeModel: ObservableObject {
             return theme
         } catch {
             print(error)
-            try filemanager.removeItem(at: url)
             return nil
         }
     }
 
-    /// Loads all available themes from `~/Library/Application Support/CodeEdit/themes/`
+    /// Loads all available themes from `~/Library/Application Support/CodeEdit/Themes/`
     ///
     /// If no themes are available, it will create a default theme and save
     /// it to the location mentioned above.
     ///
-    /// When overrides are found in `~/Library/Application Support/CodeEdit/.settings.json`
+    /// When overrides are found in `~/Library/Application Support/CodeEdit/settings.json`
     /// they are applied to the loaded themes without altering the original
-    /// the files in `~/Library/Application Support/CodeEdit/themes/`.
+    /// the files in `~/Library/Application Support/CodeEdit/Themes/`.
     func loadThemes() throws { // swiftlint:disable:this function_body_length
```

Comment:
It would be great if you could split this function up as it currently doesn't follow the single responsibility principle.

Reply:
I will do this in a follow-up PR when we re-enable theme editing